molecule/resources/playbooks/prepare.yml

```diff
@@ -25,3 +25,24 @@
     ansible.builtin.service:
       name: ipa
       state: started
+
+  - name: Wait for Kerberos KDC to be started.
+    ansible.builtin.systemd:
```

Comment:
This is trying to start the service if it is not running. It might be better to use something like `ansible.builtin.service_facts`.

Reply:
Fixed, using service_facts.